docs/wcs/index.rst

```diff
@@ -138,8 +154,8 @@ Supported projections
 
 As `astropy.wcs` is based on `wcslib`_, it supports the standard
 projections defined in the WCS papers.  These projection codes are
-specified in the second part of the ``CUNITn`` keywords (accessible
-through `Wcsprm.cunit <astropy.wcs.Wcsprm.cunit>`), for example,
+specified in the second part of the ``CTYPEn`` keywords (accessible
+through `Wcsprm.ctype <astropy.wcs.Wcsprm.ctype>`), for example,
 ``RA-TAN-SIP``.  The supported projection codes are:
```

Comment:
Is the SIP ending in wcslib now (it's not in the standard) or is it in pywcs only?
This should be ``RA---TAN-SIP`` (2 more hyphens).